src/GraphQL/Utilities/SchemaBuilder.cs

```diff
@@ -225,16 +239,19 @@ protected virtual FieldType ToFieldType(string parentTypeName, GraphQLFieldDefin
 
             var args = fieldDef.Arguments.Select(ToArguments);
             field.Arguments = new QueryArguments(args);
+            field.DeprecationReason = fieldConfig.DeprecationReason;
```

Comment:
`ToSubscriptionFieldType` also should have this line. Is not it so?

Reply:
It should, thanks!